packages/fscomponents/src/components/MultiCarousel/MultiCarousel.web.tsx

```diff
@@ -107,6 +107,8 @@ export const MultiCarousel = <ItemT, >(props: MultiCarouselProps<ItemT>) => {
     hideScrollbar,
     itemStyle,
     itemsAreEqual,
+    autoplay = true,
```

Comment:
Did the previous version of this carousel have some form of autoplay by default? If not then we should default this to false.

Reply:
Having this not as the default will require additional submarine changes so that the value can be set.
Do we want to do that instead?